main/getting-started/deploying.md

```diff
@@ -1,63 +1,91 @@
+The contract deployment process uses [`zoe.install()`](https://agoric.com/documentation/zoe/api/zoe.html#e-zoe-install-code-moduleformat) 
+to install the contract source code on-chain. This returns an *installation handle* associated with the 
 source code. In a typical contract deployment, the installation handle is stored 
-in the default shared registry so that it is broadly accessible on the chain.
-Its registry key is written to a config file in the `ui` directory. 
+in the default shared registry so it is broadly accessible on the chain.
+Its registry key is written to a config file in the Dapp's `ui` directory as shown below.
 
-See the [`dapp-encouragement` repository](https://github.com/Agoric/dapp-encouragement/blob/master/contract/deploy.js) for an example of a typical contract deploy script.
+By default, when you run `agoric initi`, your Dapp gets 
+the [`dapp-encouragement` `contract/deploy.js` file](https://github.com/Agoric/dapp-encouragement/blob/master/contract/deploy.js), 
+which is our example of a typical contract deploy script.
 
-Deploying the `dapp-encouragement` contract (e.g., with `agoric deploy contract/deploy.js`) installs it on chain, and generates the 
-file `./ui/public/conf/installationConstants.js`
-with contents like:
+Deploying the `dapp-encouragement` contract (e.g., with `agoric deploy contract/deploy.js` after `agoric init` 
+copied it into a local directory) installs it on chain, and generates the 
+file `./ui/public/conf/installationConstants.js`with contents like:
 ```js
 // GENERATED FROM dapp-encouragement/contract/deploy.js
 export default {
     "CONTRACT_NAME": "encouragement",
     "INSTALLATION_REG_KEY": "encouragementinstallation_9794"
 };
 ```
-The Registry determines the particular registry key, in this case
+The Registry determines the installation registry key, in this case
 `"encouragementinstallation_9794"`, so it is different for each contract deployment.
 
 ## Application service deployment and setup
 
-You will need to customize the API server deployment and setup much more
-for your particular application and 
-contract. Some dapps use a singleton contract instance that must be created, 
-find and connect to on-chain resources like issuers for specific currencies, 
-create new on-chain resources like new currencies or NFTs, create new Purses
-for the application's use, or seed the on-chain contract with initial orders
-or configuration.
-
-The `api/deploy.js` scripts from the example contracts can show some of the 
-range of these custom setup actions:
+Next, let's look at *application deployment and setup*. As compared to contract deployment, 
+you need to customize the API server deployment and setup much more
+for your particular application and contract. Some Dapps use a singleton contract instance **tyg todo: Could use a succinct defintion of "singleton" here. I think it means one basic contract that only interacts with itself, but aren't sure about that** that potentially must:
```

Comment:
A singleton contract would be like our multipoolAutoswap, which presumes that it will be installed once, and serve all customers, as opposed to an auction or swap contract, which would be installed once, but instantiated separately for each sale it is being used to manage.